src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -422,7 +425,7 @@
   "Parses given json (a string or a reader) into a map of paths to types, i.e. `{[\"bob\"} String}`.
 
   Uses Jackson Streaming API to skip allocating data structures, eschews allocating values when possible.
-  Respects *nested-field-column-max-row-length*."
+  Respects *nested-field-columns-max-row-length*."
```

Comment:
Perhaps make this a [[reference]]

Reply:
9b04fb0ca9d8d98a636088162cdbc4eb3f319e78